tensorrt_llm/_torch/models/modeling_qwen3_next.py

```
@@ -1227,6 +614,165 @@ def forward(
+                hidden_states = self.get_last_token_states(
+                    hidden_states, attn_metadata)
+            else:
+                hidden_states = hidden_states[-1].unsqueeze(0)
+
+        enable_attention_dp = self.model_config.mapping.enable_attention_dp
+        enable_lm_head_tp_in_adp = enable_attention_dp and self.model_config.mapping.enable_lm_head_tp_in_adp
+
+        if enable_lm_head_tp_in_adp:
+            self.mapping_lm_head_tp = create_lm_head_tp_mapping(
+                self.model_config.mapping, hidden_states.shape[0])
+            hidden_states = allgather(hidden_states,
```

Flake8 (7.3.0)

[error] undefined name 'allgather'

(F821)